src/essentials/Colors/Colors.ts

```diff
@@ -63,27 +63,27 @@ export const SemanticColors = {
     transparent: Colors.transparent,
     foreground: {
         primary: Colors.blue.primary[900],
+        accent: {
+            default: Colors.blue.secondary[900],
+            emphasized: Colors.blue.secondary[1000]
+        },
+        neutral: {
+            faded: Colors.blue.primary[200],
+            default: Colors.blue.primary[350],
+            emphasized: Colors.blue.primary[550]
+        },
         'on-background': {
             primary: Colors.white,
             accent: Colors.white,
+            neutral: Colors.blue.primary[900],
+            disabled: Colors.white,
             info: Colors.white,
             success: Colors.white,
             warning: Colors.blue.primary[900],
-            danger: Colors.white,
```

Comment:
Why are we deleting tokens?

Reply:
I'm not deleting them, just moving them around so all have the same order (in order for the tables in each scheme to match)